README.md

```diff
@@ -29,11 +29,14 @@ USAGE:
    img2lambda [options]
 
 GLOBAL OPTIONS:
-   --image value, -i value             Name of the source container image. For example, 'my-docker-image:latest'
+   --image value, -i value             Name of the source container image. For example, 'my-docker-image:latest'. The Docker daemon must be pulled locally already.
```

Comment:
Correction: The Docker *image* must be pulled locally already.

Reply:
I had copied this from the output of the actual binary. This mistake was also present in main.go which I have now fixed.